### ISSUE TITLE
Can't configure library for multi-tenant

### ISSUE THREAD

**Issue by jnjdev42** [auth0/react-native-auth0#911]
### Checklist

- [X] The issue can be reproduced in the [react-native-auth0 sample app](https://github.com/auth0-samples/auth0-react-native-sample) (or N/A).
- [X] I have looked into the [Readme](https://github.com/auth0/react-native-auth0#readme), [Examples](https://github.com/auth0/react-native-auth0/blob/master/EXAMPLES.md), and [FAQ](https://github.com/auth0/react-native-auth0/blob/master/FAQ.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/react-native-auth0/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/react-native-auth0/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

I followed auth0 documentation and it seems we can't configure the library for multi tenant with mulmti domain and multi redirect uri.

### Reproduction

1. Configure auth0 with adding this line in build.gradle :  
2. manifestPlaceholders = [auth0Domain: "domain", auth0Scheme: "${applicationId}.auth0"]

### Additional context

I tried to remove manifestPlaceholders to add multiple data intent filter into manifest. But, application not compiling. Because it needs manifestPlaceholders "auth0domain" to compile.

### react-native-auth0 version

v3.2.1

### React Native version

0.74.1

### Platform

Android

### Platform version(s)

14

**Comment by brth31** [comment]
@jnjdev42 this is a potential duplicate of https://github.com/auth0/react-native-auth0/issues/747

**Comment by jnjdev42** [comment]
> @jnjdev42 this is a potential duplicate of #747

No. The problem for me is about redirections after login and after logout. Moreover, the problem is to set "static" value inside gradle for a domain. Which is not what I am looking for, as I work with different domains.

**Comment by poovamraj** [comment]
Hey @jnjdev42, you want to use multiple domains for redirection if I understand this issue, have you considered a setup like this - https://github.com/auth0/react-native-auth0/issues/747#issuecomment-2439196254

Or is your callback URL auto-generated so that you can't hardcode them?
